doc/source/whatsnew/v0.17.1.txt

```diff
@@ -96,7 +96,7 @@ Bug Fixes
 
 
 
-
+- Bug in ``DataFrame.pct_change()`` was not propagating axis on fillna method (:issue:`11150`)
```

Comment:
use double backticks around ``axis`` (and say ``axis`` keyword) and 'on ``.fillna()`` method'